components/layouts/page/page.js

```diff
@@ -1,23 +1,23 @@
 /**
- * Layout - Page
+ * Layout — Page
  * Allows us to specify some rules for how the page will be rendered
  */
 
 // React
 import { bool, node, object, shape, string } from 'prop-types'
 
 // Style
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 
 // UI
 import { Container, MetaHead, PageHeading } from '../../'
 
 // Config
 import { Brand, Canonical } from 'config'
 
-export const Page = ({ children, fluid, meta, pageHeading }) => {
+export const Page = ({ children, fluid, marginTop, meta, pageHeading }) => {
   return (
-    <StyledPage>
+    <StyledPage marginTop={marginTop}>
```

Comment:
@zemqi I removed the margins from here in favour of using the Spacer later - also I would prefer not to use MarginTop - please remove for now.

Reply:
How to up page and compare with transparent Navbar (for example), as I did win DryKISS to compare borders of logo and hero. Add negative margins to Navbar is better?

Reply:
@zemqi Just remove for now I will solve this later, I have a better plan for the layouts